addons/google_gmail/models/google_gmail_mixin.py

```diff
@@ -72,7 +72,7 @@ def open_google_gmail_uri(self):
         if not self.google_gmail_uri:
             raise UserError(_('Please configure your Gmail credentials.'))
 
-        if not self.smtp_user:
+        if not self.user:
```

Comment:
```suggestion
        if "smtp_user" in self._fields and not self.smtp_user:
```
It is smpt_user for outgoing user, but the fetchmail server uses `user` for the username. So checking if "smtp_user" exists in fields before checking if it is set should work

Reply:
thanks, updated @jorv-odoo